ipatests/pytest_ipa/integration/fixtures.py

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
pytest fixtures are usually stored in a ``conftest.py``, which is auto-imported by pytest.

Reply:
fixed

Reply:
This approach is applicable for fixtures which are applicable for all tests. Here this fixture is specific to a particular set of tests, so IMO its better to have this in that test class's install class method.

Reply:
This is ipa_ad_trust fixture. It can be used to establish trust between IPA and AD when no special options are required. So this fixture could be used in this module, in test_smb, test_sssd and any new test suite which needs to setup trust.

Reply:
@wladich 
Still this fixture servers the trust scenario only.
I think we can move this fixture to a shared file(e.g pytest_ipa/integreation/tasks.py) and can import/use where ever necessary.

Reply:
@kaleemsiddiqu Please see comment https://github.com/freeipa/freeipa/pull/4614#discussion_r417246592

Personally I do not have any preference where to store this small fixture - in a separate file or in conftest or somewhere else.

Reply:
I only want to say that scope of fixtures defined in conftest.py is very large and applies to almost every tests and thats when we should define it. IMO ipa_trust_ad does not fit that case. Its upto you and @tiran to decide, i just shared my opinion.